03/app.js

```diff
@@ -6,13 +6,65 @@ const b = randomNumber(min, max);
 const c = randomNumber(min, max);
 
 console.log(a, b, c);
+const sum = getSum(a, b, c);
+const even = isEven(sum);
 
 
+showInfo(sum, even);
 
+function randomNumber(min, max) {
+    return Math.round((Math.random() * (max - min)) + min);
+}
 
 
 
 
-function randomNumber(min, max) {
-    return Math.round((Math.random() * (max - min)) + min);
-}
\ No newline at end of file
+// getSum 
+function getSum(par1, par2, par3) {
+    const arr = [];
+    
+   arr.push(par1);
+   arr.push(par2);
+   arr.push(par3);
+
+   const sortedArray= arr.sort(function(a, b) {
+       return a - b;
+   });
+
+   const largest = Math.max(...sortedArray);
+   const nextLargest = sortedArray[sortedArray.length -2];
```

Comment:
Skoro masz już posortowane to nie łatwiej pobrać ostatnią i przed ostatnią? 
`Math.max()` w tym wypadku uważam, za zbędne ;)